Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -186,11 +202,31 @@ namespace AzToolsFramework
             }
             else // child must be a label or a PropertyEditor
             {
-                // todo: handle this when PropertyHandlerWidgets are done, note that a PropertyEditor could theoretically be a QLabel too...
+                // pare down the path to this node, then look up and set the value from the DOM
+                auto subPath = fullPath;
+                for (size_t pathEntryIndex = fullPath.size() - 1; pathEntryIndex > pathIndex; --pathEntryIndex)
+                {
+                    subPath.Pop();
+                }
```

Comment:
Another note to mostly myself: this is another potential use-case for an `AZ::Dom::PathView`